src/runner/src/bin/main.rs

```diff
@@ -1,10 +1,27 @@
 use structopt::StructOpt;
-use fluvio_future::task::run_block_on;
 use fluvio_run::RunCmd;
+use tracing_subscriber::{Registry, prelude::*};
 
 fn main() -> Result<(), Box<dyn std::error::Error>> {
-    fluvio_future::subscriber::init_tracer(None);
-    let cmd = RunCmd::from_args();
-    run_block_on(cmd.process())?;
+    let cmd: RunCmd = RunCmd::from_args();
+
+    let service_name = match &cmd {
+        RunCmd::SC(_) => "fluvio-sc",
+        RunCmd::SPU(_) => "fluvio-spu",
+        _ => "fluvio-run",
+    };
+
+    let (tracer, _uninstall) = opentelemetry_jaeger::new_pipeline()
+        .with_service_name(service_name)
+        .install()
+        .unwrap();
+
+    Registry::default()
+        .with(tracing_subscriber::EnvFilter::new("DEBUG"))
+        .with(tracing_subscriber::fmt::layer())
+        .with(tracing_opentelemetry::layer().with_tracer(tracer))
```

Comment:
make this as optional configuration.

Reply:
Hmm I am trying, but it seems like it is not working when I make things optional, even if the feature is enabled.

Reply:
Just figured out the solution to this, PR is updated